File: src/main/java/com/github/weewar/mapviewer/exceptions/ImageResizeException.java
package com.github.weewar.mapviewer.exceptions;

public class ImageResizeException extends RuntimeException {
    public ImageResizeException() {
        super();
    }

    public ImageResizeException(String message) {
        super(message);
    }

    public ImageResizeException(String message, Throwable cause) {
        super(message, cause);
    }

    public ImageResizeException(Throwable cause) {
        super(cause);
    }
}

File: src/main/kotlin/com/github/weewar/mapviewer/exceptions/ImageResizeException.kt
package com.github.weewar.mapviewer.exceptions

class ImageResizeException : RuntimeException {
    constructor() : super() {}

    constructor(message: String) : super(message) {}

    constructor(message: String, cause: Throwable) : super(message, cause) {}

    constructor(cause: Throwable) : super(cause) {}
}
